### PR TITLE
Updated translations with latest Crowdin changes

### PR DESCRIPTION
**Pull request by ssddanbrown**

### CODE REVIEW

File: lang/fr/activities.php [hunk 1]
```diff
@@ -84,6 +84,14 @@
     'webhook_delete' => 'supprimer un Webhook',
     'webhook_delete_notification' => 'Webhook supprimé avec succès',
 
+    // Imports
+    'import_create' => 'created import',
```
Comment (ORN-Fox) [inline lang/fr/activities.php:88]: ℹ️  The translations are not in French

File: lang/fr/entities.php [hunk 2]
```diff
@@ -39,9 +39,30 @@
...
+    'import_run' => 'Run Import',
+    'import_size' => ':size Import ZIP Size',
+    'import_uploaded_at' => 'Uploaded :relativeTime',
+    'import_uploaded_by' => 'Uploaded by',
+    'import_location' => 'Import Location',
+    'import_location_desc' => 'Select a target location for your imported content. You\'ll need the relevant permissions to create within the location you choose.',
+    'import_delete_confirm' => 'Are you sure you want to delete this import?',
+    'import_delete_desc' => 'This will delete the uploaded import ZIP file, and cannot be undone.',
+    'import_errors' => 'Import Errors',
+    'import_errors_desc' => 'The follow errors occurred during the import attempt:',
```
Comment (ORN-Fox) [inline lang/fr/entities.php:65]: ℹ️  The translations are not in French

File: lang/fr/errors.php [hunk 3]
```diff
@@ -105,6 +105,18 @@
...
+    'import_zip_cant_decode_data' => 'Could not find and decode ZIP data.json content.',
+    'import_zip_no_data' => 'ZIP file data has no expected book, chapter or page content.',
+    'import_validation_failed' => 'Import ZIP failed to validate with errors:',
+    'import_zip_failed_notification' => 'Failed to import ZIP file.',
+    'import_perms_books' => 'You are lacking the required permissions to create books.',
+    'import_perms_chapters' => 'You are lacking the required permissions to create chapters.',
+    'import_perms_pages' => 'You are lacking the required permissions to create pages.',
+    'import_perms_images' => 'You are lacking the required permissions to create images.',
+    'import_perms_attachments' => 'You are lacking the required permission to create attachments.',
+
```
Comment (ORN-Fox) [inline lang/fr/errors.php:119]: ℹ️  The translations are not in French

File: lang/fr/settings.php [hunk 4]
```diff
@@ -162,6 +162,7 @@
     'role_access_api' => 'Accès à l\'API du système',
     'role_manage_settings' => 'Gérer les préférences de l\'application',
     'role_export_content' => 'Exporter le contenu',
+    'role_import_content' => 'Import content',
```
Comment (ORN-Fox) [inline lang/fr/settings.php:165]: ℹ️  The translations are not in French

File: lang/fr/validation.php [hunk 5]
```diff
@@ -105,6 +105,11 @@
     'url'                  => ':attribute a un format invalide.',
     'uploaded'             => 'Le fichier n\'a pas pu être envoyé. Le serveur peut ne pas accepter des fichiers de cette taille.',
 
+    'zip_file' => 'The :attribute needs to reference a file within the ZIP.',
+    'zip_file_mime' => 'The :attribute needs to reference a file of type :validTypes, found :foundType.',
+    'zip_model_expected' => 'Data object expected but ":type" found.',
+    'zip_unique' => 'The :attribute must be unique for the object type within the ZIP.',
+
```
Comment (ORN-Fox) [inline lang/fr/validation.php:112]: ℹ️  The translations are not in French

**Review by ORN-Fox — COMMENTED** [review]
ℹ️ French translations are not in French

**Comment by ssddanbrown** [comment]
@ORN-Fox Newly added translations will always be in English initially, until they're translated.
This is an expected part of the process, please don't flag these. 

If you want to help translate these, and you're not already helping with that, you can follow the [translations](https://github.com/BookStackApp/BookStack?tab=readme-ov-file#-translations) part of the readme to get started.

**Comment by ORN-Fox** [comment]
@ssddanbrown I understand better, it was not specified in the readme.

**Pull request merged**